source/core.py

```diff
@@ -132,6 +132,13 @@ def restart(disableAddons=False, debugLogging=False):
 		sys.argv.remove("--ease-of-access")
 	except ValueError:
 		pass
+	if globalVars.appArgs.cmdLineLanguage:

```

Comment:
I think cmdLanguage can just be language. globalVars.appArgs already implies that we're dealing with a command line argument here. Having said that, I note that some magic is happening with the configPath, but that could be considered confusing.

Reply:
> I think cmdLanguage can just be language. globalVars.appArgs already implies that we're dealing with a command line argument here.

If i'm not wrong, other members of `globalVars.appArgs` convey their default value if not specified from the command line. This one is different as, if not specified on the command line, it would not be set to the value in the configuration. Thus, I chose that name to be extra explicit and ensure no one mistakes the intent behind this member.
Let me know if you feel like my reasoning is wrong here and I will rename it as you suggest.

> Having said that, I note that some magic is happening with the configPath, but that could be considered confusing.

I don't get your point here. Could you please elaborate?

Reply:
> > Having said that, I note that some magic is happening with the configPath, but that could be considered confusing.
> 
> I don't get your point here. Could you please elaborate?

If you do not start NVDA with the --config-path parameter, globalVars.appArgs.configPath is still populated with the actual config path. It is also widely used throughout the code base to refer to the config path. A similar thing could happen with globalVars.appArgs.language in that if it is not overridden using the command line, can be set to the actual language.

> Just a suggestion: Isn't it possible to show up a simple question on startup if NVDA was started with a command line flag?

I think this suggestion makes sense, as it will address the concern about what is displayed in the general settings panel for the language.